spec/configuration_spec.lua

```diff
@@ -125,6 +125,39 @@ describe('Configuration object', function()
       assert.same(services, filter_services(services, { '42' }))
       assert.same({}, filter_services(services, { '21' }))
     end)
+
+    describe("with service filter", function()
+        local mockservices = {
+          {id=42, backend={endpoint="test.foo.com"}},
+          {id=12, backend={endpoint="staging.foo.com"}},
+          {id=21, backend={endpoint="prod.foo.com"}},
+        }
+
+      it("default case", function()
```

Comment:
We should try to choose a description here that forms a correct sentence when combined with the `describe` above.
So instead of `with service filter default case` we can have something like `with service filter does something`.
This is helpful when trying to understand why a particular test case fails.